modules/core/arrow-data/src/test/kotlin/arrow/data/ValidatedTest.kt

```diff
@@ -177,66 +169,71 @@ class ValidatedTest : UnitSpec() {
             Invalid(10).withEither { it } shouldBe Invalid(10)
         }
 
-        "Cartesian builder should build products over homogeneous Validated" {
-            Validated.applicative<String>().map(
-                    Valid("11th"),
-                    Valid("Doctor"),
-                    Valid("Who"),
-                    { (a, b, c) -> "$a $b $c" }) shouldBe Valid("11th Doctor Who")
-        }
+        with (VAL_AP) {
 
-        "Cartesian builder should build products over heterogeneous Validated" {
-            Validated.applicative<String>().map(
-                    Valid(13),
-                    Valid("Doctor"),
-                    Valid(false),
-                    { (a, b, c) -> "${a}th $b is $c" }) shouldBe Valid("13th Doctor is false")
-        }
+            "Cartesian builder should build products over homogeneous Validated" {
+                VAL_AP.map(
```

Comment:
I believe you are here already in the scope of the applicative and therefore don't need to prefix `<VAL_AP>.map`

Reply:
Tru